cluster/gce/config-test.sh

```diff
@@ -43,7 +43,7 @@ PREEMPTIBLE_NODE=${PREEMPTIBLE_NODE:-false}
 PREEMPTIBLE_MASTER=${PREEMPTIBLE_MASTER:-false}
 KUBE_DELETE_NODES=${KUBE_DELETE_NODES:-true}
 KUBE_DELETE_NETWORK=${KUBE_DELETE_NETWORK:-true}
-CREATE_CUSTOM_NETWORK=${CREATE_CUSTOM_NETWORK:-false}
+CREATE_CUSTOM_NETWORK=${CREATE_CUSTOM_NETWORK:-true}
```

Comment:
I'm not sure I understand why you're changing this default?

Reply:
They are in the `[REMOVE ME]` commit. I added this to prove my changes fixes the issue :)

Reply:
Sorry, missed that.